spec/integration/outputs/transport_create_spec.rb

```diff
@@ -0,0 +1,94 @@
+require "spec/es_spec_helper"
+
+describe "transport client create actions" do
+  require "logstash/outputs/elasticsearch"
+  require "elasticsearch"
+
+  def get_es_output(action, id = nil)
```

Comment:
it is very specific to transport client test.

I will go ahead and add a similar function to the helper to create a new plugin instance